### PR TITLE
Expand section 2 - Concepts.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Part of ongoing implementation of https://github.com/solid/specification/issues/74

### CODE REVIEW

File: oidc-authentication.md [hunk 1]
```diff
@@ -241,10 +241,119 @@ As defined by [OIDC]:
...
+used by the object capabilities community.
+
+Although authorization is largely out of scope for this document, it is worth
+mentioning that this specification aims to support this hybrid approach, and
+provide functionality for authentication as well as token-based capabilities.
 
 ## 2.2 Identity
 
+For a given [entity]() (whether a person, an organization or a software agent),
+the term [identity]() refers to a combination of: an [identifier]() (such as a
```
Comment (jaxoncreed) [inline oidc-authentication.md]: What would entity and identity link to?

File: oidc-authentication.md [hunk 2]
```diff
@@ -241,10 +241,119 @@ As defined by [OIDC]:
...
+There are many different kinds of software that we’re including in the broad
+term [client]().
+
+**We can classify clients by:**
+
+- By deployment type: Server-side web apps, in-browser JS web apps, mobile
+  apps, desktop apps, constrained apps and IoT devices.
+- Clients that have their own identity (bots/services) vs clients that are
+  always “piloted” by a user (apps)
+- “Attended” vs “Unattended” clients
```
Comment (jaxoncreed) [inline oidc-authentication.md]: Could you provide a description of what his is? Is this not a repeat of the bullet above?

File: oidc-authentication.md (hunk 1 above)
Comment (dmitrizagidulin) [reply oidc-authentication.md]: When we convert to Bikeshed, all these empty links will auto-link to the definitions in the Terminology section.

File: oidc-authentication.md (hunk 2 above)
Comment (dmitrizagidulin) [reply oidc-authentication.md]: Sure thing

**Comment by dmitrizagidulin** [comment]
Closing this PR. The main panel spec has moved in a different direction.
I've archived the original draft of the spec (in this PR) in https://github.com/interop-alliance/multi-rs-did-auth.

**Pull request closed without merging**